ynr/apps/elections/views.py

```diff
@@ -0,0 +1,1008 @@
+            "WMC:W07000074": "66107",
+            "WMC:W07000046": "66083",
+            "WMC:W07000076": "66106",
+            "WMC:N06000013": "66136",
+            "WMC:N06000012": "66135",
+            "WMC:N06000011": "66134",
+            "WMC:N06000010": "66133",
+            "WMC:N06000017": "66140",
+            "WMC:N06000016": "66139",
+            "WMC:N06000015": "66138",
+            "WMC:N06000014": "66137",
+            "WMC:W07000047": "66118",
+            "WMC:E14001024": "65769",
+            "WMC:W07000080": "66119",
+            "WMC:E14000572": "65750",
+            "WMC:E14000573": "65679",
+            "WMC:E14000570": "65981",
+            "WMC:E14000571": "65583",
+            "WMC:E14000576": "65841",
+            "WMC:E14000577": "65628",
+            "WMC:E14000574": "65805",
+            "WMC:E14000575": "65753",
+            "WMC:E14000578": "65646",
+            "WMC:E14000579": "65712",
+            "WMC:W07000079": "66114",
+            "WMC:E14000617": "65927",
+            "WMC:E14000616": "65826",
+            "WMC:E14000615": "65913",
+            "WMC:E14000614": "65906",
+            "WMC:E14000613": "66035",
+            "WMC:E14000612": "65975",
+            "WMC:E14000611": "66015",
+            "WMC:E14000610": "65708",
+            "WMC:E14000619": "65878",
+            "WMC:E14000618": "65790",
+            "WMC:W07000078": "66098",
+            "WMC:E14001062": "66037",
+            "WMC:E14001061": "65965",
+            "WMC:E14001060": "65935",
+            "WMC:E14000958": "65728",
+            "WMC:E14000959": "65942",
+            "WMC:E14000954": "65956",
+            "WMC:E14000955": "66016",
+            "WMC:E14000956": "65580",
+            "WMC:E14000957": "65876",
+            "WMC:E14000950": "65775",
+            "WMC:E14000951": "65596",
+            "WMC:E14000952": "65652",
+            "WMC:E14000953": "65678",
+            "WMC:S14000004": "14401",
+            "WMC:S14000005": "14402",
+            "WMC:S14000006": "14403",
+            "WMC:S14000007": "14404",
+            "WMC:S14000001": "14398",
+            "WMC:S14000002": "14399",
+            "WMC:S14000003": "14400",
+            "WMC:S14000008": "14405",
+            "WMC:S14000009": "14406",
+            "WMC:E14000763": "65937",
+            "WMC:E14000762": "65791",
+            "WMC:E14000761": "65925",
+            "WMC:E14000760": "65585",
+            "WMC:E14000767": "65968",
+            "WMC:E14000766": "65871",
+            "WMC:E14000765": "66026",
+            "WMC:E14000764": "65882",
+            "WMC:E14000680": "65569",
+            "WMC:E14000681": "65856",
+            "WMC:E14000769": "66001",
+            "WMC:E14000768": "65939",
+            "WMC:E14000684": "65714",
+            "WMC:E14000685": "65881",
+            "WMC:E14000686": "65756",
+            "WMC:E14000687": "65570",
+            "WMC:S14000019": "14416",
+            "WMC:S14000018": "14415",
```

Comment:
Oh god. The horror, the horror! :scream:

Reply:
To be fair to myself, this was [something I did within 3 hours of the snap general election being called, when I was in a very different timezone](b8faf2b62de21c580f401cfd39ef337bf26b9f8f). Still, yeah.